tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "target": "es2019",
     "module": "commonjs",
     "outDir": "build",
-    "alwaysStrict": true,
```

Comment:
Is this not needed anymore?

Reply:
Technically not, but could be useful to add the more verbose options for clarity. From the docs:
> Enable all strict type checking options.Enabling --strict enables --noImplicitAny, --noImplicitThis, --alwaysStrict, --strictBindCallApply, --strictNullChecks, --strictFunctionTypes and --strictPropertyInitialization.